package/rmt-server.spec

```diff
@@ -141,6 +141,20 @@ sed -i -e '/BUNDLE_PATH: .*/cBUNDLE_PATH: "\/usr\/lib64\/rmt\/vendor\/bundle\/"'
     -e 's/^BUNDLE_JOBS: .*/BUNDLE_JOBS: "1"/' \
     %{buildroot}%{app_dir}/.bundle/config
 
+# cleanup of /usr/bin/env commands
+%if 0%{?use_ruby_2_4}
+grep -rl '\/usr\/bin\/env ruby' %{buildroot}%{lib_dir}/vendor/bundle/ruby | xargs \
+    sed -i -e's@\/usr\/bin\/env ruby.ruby2\.4@\/usr\/bin\/ruby\.ruby2\.4@g' \
+    -e 's@\/usr\/bin\/env ruby@\/usr\/bin\/ruby\.ruby2\.4@g'
+%else
+grep -rl '\/usr\/bin\/env ruby' %{buildroot}%{lib_dir}/vendor/bundle/ruby | xargs \
```

Comment:
yikes :)  no way to escape that alltogether?

Reply:
The `\` is a special character and needs to be escaped in these cases.

Reply:
@tmuntan1 Do we actually need any vendored binaries? I think they can be just deleted.

Reply:
@ikapelyukhin I would have just removed them if it was only `/usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/`, but it's also `/usr/lib64/rmt/vendor/bundle/ruby/2.5.0/gems/`. I don't want to remove anything in that directory.

```
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/nokogiri /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/puma /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/pumactl /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/rackup /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/rails /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/rake /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/thor /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/yard /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/yardoc /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/bin/yri /usr/bin/env ruby.ruby2.5
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/gems/loofah-2.0.3/benchmark/benchmark.rb /usr/bin/env ruby
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/gems/mini_portile2-2.3.0/concourse/tasks/rake-test/task.sh /usr/bin/env bash
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/gems/nokogiri-1.8.1/build_all /usr/bin/env bash
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/gems/nokogiri-1.8.1/patches/sort-patches-by-date /usr/bin/env ruby
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/gems/railties-5.1.3/exe/rails /usr/bin/env ruby
[   37s] rmt-server.x86_64: E: env-script-interpreter (Badness: 9) /usr/lib64/rmt/vendor/bundle/ruby/2.5.0/gems/rake-12.0.0/exe/rake /usr/bin/env ruby